### PR TITLE
Value as sub string

### PR DESCRIPTION
**Pull request by inos-github**
functionality is checked for 
- using the magic string not at all
- using the magic string as a single expression (like it was before)
- using the magic string as a substring
- using multiple magic strings 

examples enhanced
- dev/3 showing a var assignment
- dev/4 showing multi var assignment

readme / screenshot completed

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -80,6 +80,14 @@ mosquitto_pub -t dev/1 -m hi
 mosquitto_pub -t dev/2 -m 'Hi Jane!'
 						dev/2 Hi Jane!
 						dev/2/report 111 * Hi Jane! 222 Hi Jane! 333
+
+mosquitto_pub -t dev/3 -m 'foo-bar'
+						dev/3 foo-bar
+						dev/3/report foo-bar
+
+mosquitto_pub -t dev/3 -m 'foo/bar'
```
Comment (jpmens) [inline README.md]: I think topic shouild be `dev/4`?

**Comment by jpmens** [comment]
That looks good; could you fix just that one typo please?

**Comment by inos-github** [comment]
thanks! :-)

**Comment by jpmens** [comment]
Closes #17 

Merged. Thank you very much.

**Pull request merged**